### PR TITLE
Fix race condition in integration test

### PR DESCRIPTION
**Pull request by aditya1702**
<details>
  <summary>PR Checklist</summary>
  
### PR Structure

* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `services/friendbot`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
* [ ] I've updated any docs ([developer docs](https://developers.stellar.org/api/), `.md`
  files, etc... affected by this change). Take a look in the `docs` folder for a given service,
  like [this one](https://github.com/stellar/go/tree/master/services/horizon/internal/docs).

### Release planning

* [ ] I've updated the relevant CHANGELOG ([here](services/horizon/CHANGELOG.md) for Horizon) if
  needed with deprecations, added features, breaking changes, and DB schema changes.
* [ ] I've decided if this PR requires a new major/minor version according to
  [semver](https://semver.org/), or if it's mainly a patch change. The PR is targeted at the next
  release branch if it's not a patch change.
</details>

### What

Fixes a race condition in testing ingestion-filtering flag.

Closes https://github.com/stellar/go/issues/4987

### CODE REVIEW

File: services/horizon/internal/integration/parameters_test.go [hunk 1]
```diff
@@ -2,12 +2,19 @@
...
 	"os"
 	"os/exec"
 	"path"
 	"strings"
+	"sync"
 	"testing"
+	"time"
 
 	"github.com/stellar/go/services/horizon/internal/paths"
 	"github.com/stellar/go/services/horizon/internal/simplepath"
```
Comment (2opremio) [inline services/horizon/internal/integration/parameters_test.go:20]: Can you rearrange the imports in the kosher order?

**Review by 2opremio — APPROVED** [review]

File: services/horizon/internal/integration/parameters_test.go [hunk 2]
```diff
@@ -212,6 +220,95 @@ func TestDisablePathFinding(t *testing.T) {
...
+	go func() {
+		defer wg.Done()
+		if err := w.Close(); err != nil {
+			t.Errorf("Failed to close Stdout")
+			return
+		}
+	}()
+
+	// Give some time for the goroutine to start
+	time.Sleep(time.Millisecond)
```
Comment (sreuland) [inline services/horizon/internal/integration/parameters_test.go:265]: this sleep seems mis-placed, can it be removed, the wg.Wait() should take care of allowing the syncing to happen.

**Review by sreuland — APPROVED** [review]
thanks for converging the tests into parameters_test.go also!

**Comment by 2opremio** [comment]
@aditya1702 the import ordering is wrong again. I would recommend using an IDE which sorts it automatically for you (both Goland and, I believe, VsCode should be able to do it automatically for you)

**Comment by aditya1702** [comment]
> @aditya1702 the import ordering is wrong again. I would recommend using an IDE which sorts it automatically for you (both Goland and, I believe, VsCode should be able to do it automatically for you)

@2opremio I dont see the error for the wrong import ordering. Were you looking at the latest version of the PR?

**Comment by 2opremio** [comment]
Ah, true, thanks

**Pull request merged**